domains/fame.json

```diff
@@ -0,0 +1,7 @@
+{
+    "description": "Redirection to the games side of my site.",
+    "Current site i want it to redirect to": "https://cutefame.net/games",
+    "owner": {
+        "username": "TheRealFame",
+        "email": "email@cutefame.net",
+        "twitter": "The_RealFame"
```

Comment:
File is cutoff....?

Reply:
Is there something more? I don't remember, I'm just using a template that other people with successful pulls used